### PR TITLE
Moth Food: a Taste of the Fleet

### PR DESCRIPTION
**Pull request by EOBGames**
## About The Pull Request
In this thrilling sequel to #59412, Moths get the culinary overhaul they've needed for a while.
Adds somewhere in the region of 50 new foods, all tied together by a connection to Moths and their culture, as well as around 10 new drinks.
image
As with #59412, this PR has three aims:
1. Introduce moth culture (as it exists in Common Core lore) to the game in a tangible way.
2. Give further variety to the chef and bartender, including some form of new cooking challenge for them.
3. To give some flavour (metaphorically and literally).

image
This time, I actually will include a complete list of the new foods and their recipes: https://pastebin.com/2VLfUfDx
(Edit: minor amendments: mothic pizza dough is now made with 5 milk, 2 quality oil, 5 salglu solution, 10 cornmeal and 5 flour)
As a nomadic people, moths have a wide variety of foods from various cultures. In particular, they've adopted a lot from humanity. Expect a mix of stranger and more distinctly alien foods, along with some human classics that have underwent a mothic transformation. Additionally, any language fans may enjoy the comments on each Mothic named food item, giving a glimpse into Mothic as a language.
image
Töchtaüse Berries
image
A popular food on the Mothic Fleet, töchtaüse berries are well known for being deeply unpleasant while raw: they're spicy, bitter, and filled with a mild poison that causes itching. Once heated to a sufficient temperature and mixed with sugar, however, they're a complex flavour that adds nicely to both sweet and savoury dishes.

Psst. Hey. Secret extra feature for secret fans:
image
This charming poster, originally by Capsandi, from #51867. Moffuchi's may be gone, but it's not forgotten.

## Why It's Good For The Game
The lizard food PR was a good first step in making Common Core lore better integrated with the game, but it was only that: a start. This is a continuation that should keep that trend going and making the universe deeper.

## Changelog

:cl:
add: The release of the new cookbook, "Mothic Cuisine: of Fleet and Flotilla" has brought Mothic food to the masses! Try out some new treats, like cheesecake balls, cheese soup, and five-cheese pizza at the kitchen today! There might even be some foods that don't have cheese, too!
add: To coincide with the new popularity of Mothic cuisine, Nanotrasen now stocks seeds for Töchtaüse Berries, a common ingredient in moth cooking, in the Megaseed vendors. Additionally, herbs and bell peppers have also made their way to the station's stock. Botanists, you know what to do.
/:cl:

### CODE REVIEW

**Comment by CRITAWAKETS** [comment]
also you added cheese curds but you didnt add poutine, missed opportunity if i say so myself

**Comment by EOBGames** [comment]
> also you added cheese curds but you didnt add poutine, missed opportunity if i say so myself

Sorry, I don't recognise the existence of the Quebecois. Not Quebec, it exists, just the Quebecois.

**Comment by CRITAWAKETS** [comment]
> > also you added cheese curds but you didnt add poutine, missed opportunity if i say so myself
> 
> Sorry, I don't recognise the existence of the Quebecois. Not Quebec, it exists, just the Quebecois.

you absolute buffoon, dont you fill silly, even a little stupid

I EXIST
therefore
THE QUEBECOIS EXIST
you can scream now

**Comment by CRITAWAKETS** [comment]
> c'eci n'est pas un canadien

thats it, flooding your discord DMs with quebecois insults

**Comment by JohnFulpWillard** [comment]
> 
> 
> also you added cheese curds but you didnt add poutine, missed opportunity if i say so myself

Don't we already have poutine?

**Comment by Zytolg** [comment]
> > also you added cheese curds but you didnt add poutine, missed opportunity if i say so myself

Only he totally did. It's just not called Poutine.

File: code/game/machinery/computer/chef_orders/order_datum.dm [hunk 1]
```diff
@@ -231,3 +266,9 @@
 	category_index = CATEGORY_SAUCES_REAGENTS
 	item_instance = /obj/item/reagent_containers/food/condiment/quality_oil
 	cost_per_order = 120 //Extra Virgin, just like you, the reader
+
+/datum/orderable_item/yoghurt
+	name = "Yoghurt"
```
Comment (necromanceranne) [inline code/game/machinery/computer/chef_orders/order_datum.dm:271]: Is this meant to be here twice?

**Comment by necromanceranne** [comment]
> > also you added cheese curds but you didnt add poutine, missed opportunity if i say so myself
> 
> Don't we already have poutine?

I'm pretty sure someone removed it or something.

**Comment by necromanceranne** [comment]
Anyway I'm just happy we finally have bell peppers

**Comment by Fryzzie** [comment]
I am disappointed there is no pizza with socks on it.

**Review by Mothblocks — CHANGES_REQUESTED** [review]
Revert your changes of rust_g.dll, you made them by mistake.

Close Dream Daemon when doing so, that is why you are getting permission errors.

**Comment by carshalash** [comment]
Uhh, dude, you forgot to change the drink qualities. Baselines are fantastic while the cocktails are nice.

**Comment by EOBGames** [comment]
> Uhh, dude, you forgot to change the drink qualities. Baselines are fantastic while the cocktails are nice.

Resolved. Curacao is no longer the best drink on station, it's only the best in real life now.

**Comment by ExcessiveUseOfCobblestone** [comment]
when you say 50 new food items are you saying you added 50 new moth exclusive food items?

**Comment by Farquaar** [comment]
I don't know. I expected their food to look more "insect-y". This just looks like human food with alien ingredients..

**Comment by EOBGames** [comment]
> when you say 50 new food items are you saying you added 50 new **moth exclusive** food items?

I added around 50 new food items, of which the vast majority are also fine for the other species. There are only two which are explicitly "moth exclusive", because they use cotton and are therefore under the cloth foodgroup. About the only thing I'd say is that lizards won't get much mileage out of this food given how dairy and grain based it is, but given they just got the lizard food PR a few months back (which was similar to this in that not many of the foods were lizard-only) I don't think they'll be hurting for it.

**Comment by EOBGames** [comment]
the merge conflict did not, in fact, go away

**Comment by EOBGames** [comment]
this PR is killing me, it will return but this particular branch is a lost cause

sorry to anyone who was excited but I can't do this anymore

**Pull request closed without merging**